### DISCUSSION TITLE
features/tag

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#16] [Announcements]
# features/tag

你可以在 VanBlog 的管理后台中轻松对分类和标签进行管理。 分类管理 分类管理位于后台的 站点管理/数据管理/分类管理: 分类管理 你可以: 新建分类; 查看分类: 可直接跳转到博客的对应页面查看相关分类的文章。; 修改分类: 对分类进行重命名，这会导致该分类下的所有文章都被移动到新的分类。; 修改分类 加密分类; 加密某个分类后，此分类下的所有...

https://vanblog.mereith.com/features/tag.html

**Comment by trainee315** [comment]
标签应该如何编辑，我没有找到。

**Comment by trainee315** [comment]
标签管理应该如何新增标签?

**Reply by Mereithhh** [reply]
不需要新增，每次新建文章或草稿，输入一个新标签自动会新建

**Comment by i-am-yimin** [comment]
分类新增但是无法看见
